From akpm@osdl.org Thu Jan 19 23:59:29 2006
Message-Id: <200601200759.k0K7xQni020724@shell0.pdx.osdl.net>
From: Andrew Morton <akpm@osdl.org>
Subject: USB: yealink printk warning fix
To: greg@kroah.com
Cc: akpm@osdl.org
From: akpm@osdl.org
Date: Thu, 19 Jan 2006 23:59:12 -0800


From: Andrew Morton <akpm@osdl.org>

drivers/usb/input/yealink.c: In function `usb_probe':
drivers/usb/input/yealink.c:910: warning: int format, different type arg (arg 4)


Signed-off-by: Andrew Morton <akpm@osdl.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/input/yealink.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/usb/input/yealink.c
+++ gregkh-2.6/drivers/usb/input/yealink.c
@@ -907,7 +907,7 @@ static int usb_probe(struct usb_interfac
 	pipe = usb_rcvintpipe(udev, endpoint->bEndpointAddress);
 	ret = usb_maxpacket(udev, pipe, usb_pipeout(pipe));
 	if (ret != USB_PKT_LEN)
-		err("invalid payload size %d, expected %d", ret, USB_PKT_LEN);
+		err("invalid payload size %d, expected %zd", ret, USB_PKT_LEN);
 
 	/* initialise irq urb */
 	usb_fill_int_urb(yld->urb_irq, udev, pipe, yld->irq_data,
